File: Programmers/pro_43165_타겟넘버.py
def getTarget(numbers,target,cnt,acc):    
    if(cnt==len(numbers)):
        if(target==acc):
            return 1
        return 0
    else:
        return (getTarget(numbers,target,cnt+1,acc+numbers[cnt])
                + getTarget(numbers,target,cnt+1,acc-numbers[cnt]))
        
def solution(numbers, target):
    return getTarget(numbers,target,0,0)

File: Programmers/pro_43165_타겟넘버.js
function solution(numbers, target) {
    const len = numbers.length
    const getTarget = (num, cnt) => {
        if (cnt == len)
            return num == target ? 1 : 0
        return getTarget(num - numbers[cnt], cnt + 1) + getTarget(num + numbers[cnt], cnt + 1)
    }
    return getTarget(0, 0);
}